docs/gh_pages/docs/config.md

```diff
@@ -0,0 +1,42 @@
+# Pebblo Configuration File
+
+### Introduction
+
+This configuration file specifies settings for various components of the Pebblo.
+
+### Configuration Details
+
+#### Daemon
+
+- `port`: Specifies the port number on which the Pebblo daemon listens for incoming connections.
+- `host`: Specifies the host address on which the Pebblo daemon is running.
```

Comment:
We should add note that if we change these values then we need to set `PEBBLO_CLASSIFIER_URL ` for PebbloSafeDataLoader to send requests to the changes host:port.